src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -18,8 +18,9 @@ namespace System.Net.WebSockets
 {
     internal sealed class WebSocketHandle
     {
-        /// <summary>Shared, lazily-initialized handler for when using default options.</summary>
-        private static SocketsHttpHandler? s_defaultHandler;
+        // Shared, lazily-initialized invokers used to avoid some allocations when using default options.
```

Comment:
I assume we already have sufficient tests for multiple websocket requests with both UseProxy false and true and the default proxy is left set?

Reply:
We already have a lot of coverage of "proxy == DefaultProxy" as that's the default all tests are exercising.

We also had a test for "proxy == null", but it was disabled against an already closed issue - re-enabled it now.